src/action-creators/alert.js

```diff
@@ -1,13 +1,21 @@
+/**
+ * @packageDocumentation
+ * @module action-creators.alert
```

Comment:
Is there a way to infer the `@module` from the directory structure? I don't like the burden of having to explicitly mark up every file when it can be generated automatically.

Reply:
Yes, there's another way to do this, we can write mapping function. See c806a04

Reply:
Great! So we can remove `@module`, but `@packageDocumentation` is still required?